rustler_codegen/src/nif.rs

```diff
@@ -0,0 +1,205 @@
+use proc_macro2::{Span, TokenStream};
+use quote::{quote, ToTokens};
+use syn::punctuated::Punctuated;
+use syn::token::Comma;
+
+pub fn transcoder_decorator(args: syn::AttributeArgs, fun: syn::ItemFn) -> TokenStream {
+    let sig = &fun.sig;
+    let name = &sig.ident;
+    let inputs = &sig.inputs;
+    let flags = schedule_flag(args.to_owned());
```

Comment:
`schedule_flag` could also use `&args` (if `extract_attr_value` did the same). This seems to work here, but maybe there is a better reason to not use references here or below?